example/ios/Podfile.lock

```diff
@@ -292,7 +292,7 @@ PODS:
     - StripeApplePay (= 22.0.0)
     - StripeCore (= 22.0.0)
     - StripeUICore (= 22.0.0)
-  - stripe-react-native (0.7.0):
+  - stripe-react-native (0.8.0):
```

Comment:
Should this be added as a release step in the future?

Reply:
eh, kinda doesn't matter, this is just the example app 🤷 I think to include this all we need is to `yarn bootstrap` again after `yarn -s --ignore-scripts publish --non-interactive --access=public`. So it'll add time but probably make things cleaner